index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>?Shalini's Portfolio</title>
+  <link rel="stylesheet" href="style.css"/>
+</head>
+<body class="body">
+  <header>
+    <nav class="head">
+      <a href="#" class="logo">My logo</a>
+  <img src="images/Union.png" class="nav-bar">
+      </nav>
+      </header>
+     <main class="container">
+        <div class="background">
+    <section class="home-section">
+      <h1 class="header">
+        I'm Shalini<br>
+        Glad to see <br>
+        you!
+    </h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product ,feature or website Look through some of my work and experience! If you like what you see and have a project you need coded ,
+        don’t hestiate to contact me.
+    </p>
+<a href="#" class="link">LET'S CONNECT</a>
+<ul class="icons">
+<li class="twitter">
+    <a href="https://twitter.com/sndprs">
+        <img src="images/Vector.png" alt="twitter icon">
+      </a>
+</li>
+<li class="linkedin">
+    <a href="https://www.linkedin.com/in/shalini-nayan-5820389/">
+        <img src="images/Linkedin icon.png" alt="linkedin icon">
+      </a>
+</li>
+  <li class="github">
+    <a href="https://github.com/ShaliniNayan">
+        <img src="images/Vector (2).png" alt="github icon">
+      </a>
+  </li>
+  <li class="medium">
+    <a href="#">
+        <img src="images/Vector (1).png" alt="medium icon">
+      </a>
+  </li>
+  <li class="angellist">
+    <a href="#">
+        <img src="images/Vector (3).png" alt="medium icon">
+      </a>
+  </li>
+   </ul>
+
+</div>
+</section>
+</main>
+
```

Comment:
- [ ] You haven't added the `works` section and the `About myself` section as per the project requirements. Kindly work on it and you should follow the project requirements as well.

![works-mobile](https://user-images.githubusercontent.com/70483835/227607210-028945e3-c6c0-4fe4-b899-ebf6a4aac579.png)
<img width="188" alt="about-myself-mobile" src="https://user-images.githubusercontent.com/70483835/227607198-fe5685d8-231f-44b4-80ba-24c7fe646129.png">